mmv1/products/networkconnectivity/Spoke.yaml

```diff
@@ -122,6 +128,10 @@ properties:
     immutable: true
     resource: 'hub'
     imports: 'name'
+  - name: 'group'
```

Comment:
Is this field updatable? If yes, can you please add the update test for this field in the [file](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/networkconnectivity/resource_network_connectivity_spoke_test.go) ? 

This is the [doc](https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test) to add an update test.

Reply:
The field is currently not updateable. That will come at a later time.

Reply:
Can you add `immutable: true` to it for this case?

Reply:
Done!